webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/services/policies/NodeConfigData.scala

```diff
@@ -508,15 +509,11 @@ ootapja6lKOaIpqp0kmmYN7gFIhp
  *
  */
 
-class TestNodeConfiguration(prefixTestResources: String = "") {
+class TestTechniqueRepo(prefixTestResources: String = ""
```

Comment:
be able to just use technique repo & git test services (and override gitRevisionProvider)